src/Script.sol

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.6.2 <0.9.0;
+pragma experimental ABIEncoderV2;
```

Comment:
Is the thinking here that we add this to `Script.sol` and `Test.sol` so users on <0.8 don't need to add it themselves? If that seems fine, and my only question is: ABI encoder v2 is the default in 0.8, so does explicitly specifying it here as experimental change anything? (i.e. like using an older/buggier version of the v2 encoder or something)

Reply:
No, I planned to remove this along with `Components`.